python/cuml/cuml/svm/linear.pyx

```diff
@@ -643,22 +644,92 @@ class LinearSVM(Base, metaclass=WithReexportedParams):
         self.classes_ = None
         self.probScale_ = None
 
+    @classmethod
+    def _get_param_names(cls):
+        return super()._get_param_names() + [
+            "penalty",
+            "loss",
+            "fit_intercept",
+            "penalized_intercept",
+            "probability",
+            "max_iter",
+            "linesearch_max_iter",
+            "lbfgs_memory",
+            "C",
+            "grad_tol",
+            "change_tol",
+            "epsilon",
+            "tol"
+        ]
+
+    @classmethod
+    def _params_from_cpu(cls, model):
+        params = {
+            "C": model.C,
+            "fit_intercept": model.fit_intercept,
+            "max_iter": model.max_iter,
+            "tol": model.tol,
+        }
+
+        return params
+
+    def _params_to_cpu(self):
+        params = {
+            "C": self.C,
+            "fit_intercept": self.fit_intercept,
+            "max_iter": self.max_iter,
+            "tol": self.grad_tol,
+        }
+
+        return params
+
+    def _attrs_from_cpu(self, model):
+        coef_ = model.coef_.reshape(1, -1)
+        return {
+            "coef_": to_gpu(coef_, order="F", dtype=np.float64),
+            "intercept_": to_gpu(model.intercept_, order="F", dtype=np.float64),
+            "n_features_in_": model.n_features_in_,
```

Comment:
```suggestion
```

This is handled by the base implementation.

Reply:
Addressed in 2c4848f28633b567381b752b2ee3bbe1b038f1ee .